bylaws/2-cf-membership.md

```diff
@@ -20,22 +16,28 @@ The following section outlines the different types of membership and the process
 
 A **project** within the CF represents collaborative innovation in open source technology. These projects are dynamic collectives of individual contributors, functioning as self-organizing units rather than formal legal entities.[^1]
 
-**Flexible and Inclusive Structure:** Projects can vary in size and scope, ranging from individual repositories to entire collections of repositories. Whether it's a single focused effort or a collection of related initiatives, each project is defined by common goals and shared assets, including code, trademarks, and domains.
+**Flexible and Inclusive Structure:** Projects vary in size and scope&mdash;a project might have a single repository, or it might encompass multiple repositories.
+Whether it's a single focused effort or a collection of related initiatives, each project is defined by common goals and shared assets, including code, trademarks, and domains.
+While the CF uses GitHub as the primary platform for collaboration, projects may be hosted on other platforms as well.
 
 [^1]: Contributors may be affiliated with various organizations or companies, but they contribute as individuals.
 
-## Project Representation
+### Project Representation
 
 **Diverse Voices, Unified Decision-Making:** Each project within the Commonhaus Foundation (CF) has its unique perspective, and its own legitimate interests in decisions taken by the foundation as a whole.
 To ensure that every project has a say in the direction of the foundation, each project appoints a representative to the [Extended Governance Committee (EGC)][egc].
-The representative is chosen according to the project's own governance practices.
-The representative is not required to have or maintain general membership of the foundation.
 
-**Dedicated Representation:** If an EGC representative is elected to the [CF Council (CFC)][cfc], we recommend appointing an additional representative for the EGC to ensure project-specific needs are represented.
+The EGC representative:
+
+- is chosen according to the project's own governance practices,
+- is not required to have or maintain general membership in the foundation, but
```

Comment:
just out of curiosity - in other foundations (e.g. apache/eclipse) all committers to a project are inherently members of the foundation (if they aren't they can't be committers and vice versa afaik).

This is afaik to at least have one foundation member be responsible/traced back to for how a commit entered the repos.

With today's automation of things one don't really need to be a committer to put code into a project - so anyone as long as the automation lets them (by mistake or process) can push code. 

...similar applies here - anyone can by this line of "is not required" influence projects/the foundation without membership.

Reply:
> all committers to a project are inherently members of the foundation

To ensure we can be member-supported, that isn't the case here. Members help by buying the foundation a beer or a fancy coffee annually (and get some special things due to that). That helps keep the lights on. I don't want to be dependent only on big donations from vendors.

The EGC is special, and I'm not requiring that person to be a (paying) regular member as they are appointed by the project and have a responsible/leadership role. They can be regular members, too, and I hope they will be, but that part isn't required.

> This is afaik to at least have one foundation member be responsible/traced back to for how a commit entered the repos.

The EGC rep is the responsible party here.

> anyone can by this line of "is not required" influence projects/the foundation without membership.

Projects are required to use the DCO. You don't have to be a member to contribute code to a project. You do have to be a member to hold an office or vote in elections for foundation officers / council members.